### PR TITLE
Feature/reusable unit

### PR DESCRIPTION
**Pull request by davidkhala**

### CODE REVIEW

File: Cargo.lock
Comment (davidkhala) [inline Cargo.lock:1]: Our practice is no lock

File: Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[workspace]
+resolver = "2"
+members = [ 
+    "example", "grpc", "http",
+    "novax-log",
+    "novax-services"
```
Comment (davidkhala) [inline Cargo.toml:6]: delete this and the published package

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # common-rs
 common library crates
+
```
Comment (davidkhala) [inline README.md:3]: No need to have this.

This typeof information is common sense.

File: example/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+[package]
+name = "example"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+novax-log = { path="../novax-log" }
+novax-http = { path="../http" }
```
Comment (davidkhala) [inline example/Cargo.toml:8]: We depends on the published package, not file

File: novax-services/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+###### Novax services
+
+1. Provide a http service stub
+2. Provide a grpc service stub
+
+
+###### Package deprecated
```
Comment (davidkhala) [inline novax-services/README.md:7]: We don't need deprecated. We need to nuke this.

File: novax-services/src/error.rs [hunk 5]
```diff
@@ -0,0 +1,19 @@
+use thiserror::Error;
+use core::result::Result as SResult;
+use tokio::sync::oneshot::error::RecvError;
+
+#[derive(Error, Debug)]
+pub enum Error {
```
Comment (davidkhala) [inline novax-services/src/error.rs:6]: Why we put this in services?

**Comment by DCh221** [comment]
Merge from feature branch to main common.rs

**Pull request merged**